packages/main/src/List.ts

```diff
@@ -233,7 +233,7 @@ type ListItemClickEventDetail = {
 
 /**
  * Fired when selection is changed by user interaction
- * in `SingleSelect`, `SingleSelectBegin`, `SingleSelectEnd` and `MultiSelect` modes.
+ * in `Single`, `SingleSelectBegin`, `SingleSelectEnd` and `Multiple` selection modes.
```

Comment:
Again.

I believe `SingleSelectBegin`, `SingleSelectEnd` should be `SingleStart`, `SingleEnd`